website/content/installation/_index.md

```diff
@@ -63,6 +63,16 @@ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifest
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/metallb.yaml
 ```
 
+{{% notice note %}}
+If you want to deploy metallb using the [experimental frr mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), apply the manifests:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/metallb-frr.yaml
+```
+
```

Comment:
```suggestion
Please do note that these manifests deploy MetalLB from the main development branch. We highly encourage cloud operators to deploy a stable released version of MetalLB on production environments!
```

Reply:
Not sure about this. Each tag is pinned to that tag's version

Reply:
Ah ignore me, this is for  the main branch